### PR TITLE
Split E2E Traffic with New LB

### PR DESCRIPTION
**Pull request by giancorderoortiz**
DO NOT MERGE!!

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   "retries": {
-    "runMode": 2
+    "runMode": 3
   },
   "videoUploadOnPasses": false,
   "env": {
     "CLIENT_ID": "mobile_android",
     "CLIENT_SECRET": "secret",
-    "API_URL": "https://40.76.109.9:9002",
+    "API_URL": "https://20.83.184.244:9002",
```
Comment (plabadie) [inline projects/storefrontapp-e2e-cypress/cypress.json:12]: I think the default url of this file should point to dev.  It was set to dev in the past to keep default traffic away from the CI backends.

File: ci-scripts/e2e-cypress.sh [hunk 2]
```diff
@@ -46,6 +50,24 @@ done
...
+    sed 's/20.83.184.244/20.83.178.185/' .env-cmdrc > .tmp && mv .tmp .env-cmdrc || true
+    
+    # modify API URL in cypress config files.
+    sed 's/20.83.184.244/20.83.178.185/' ${CYPRESS_JSON} > tmp0.json && mv tmp0.json ${CYPRESS_JSON} || true
+    sed 's/20.83.184.244/20.83.178.185/' ${CYPRESS_CI_JSON}  > tmp1.json && mv tmp1.json ${CYPRESS_CI_JSON} || true
+    sed 's/20.83.184.244/20.83.178.185/' ${CYPRESS_CI_B2B_JSON}  > tmp2.json && mv tmp2.json ${CYPRESS_CI_B2B_JSON} || true
+
+    sleep 5
+fi
+
```
Comment (plabadie) [inline ci-scripts/e2e-cypress.sh:70]: Are there some alternatives to `sed` we could use for more robustness over time?  Could some kind of environment variable be used?

Comment (giancorderoortiz) [reply ci-scripts/e2e-cypress.sh:70]: I am not aware of any. I'm open to suggestions.

File: projects/storefrontapp-e2e-cypress/cypress.json (hunk 1 above)
Comment (giancorderoortiz) [reply projects/storefrontapp-e2e-cypress/cypress.json:12]: Agree. Updated by mistake. Will revert.

File: ci-scripts/e2e-cypress.sh (hunk 2 above)
Comment (plabadie) [reply ci-scripts/e2e-cypress.sh:70]: Perhaps something can be done by overriding a config on the fly.  See "Overriding Options" https://docs.cypress.io/guides/references/configuration#Overriding-Options

**Comment by giancorderoortiz** [comment]
no longer needed

**Pull request closed without merging**